apps/system/style/notifications/notifications.css

```diff
@@ -102,7 +87,7 @@
 
 #notification-toaster.displayed {
   transform: translateY(0);
-  transition: transform .2s ease 1.2s, background-color 0s ease 1.2s, visibility 0s ease 1.2s;
+  transition: transform .2s ease .4s, background-color 0s ease, visibility 0s ease;
```

Comment:
If you're I think you're talking about https://bugzilla.mozilla.org/show_bug.cgi?id=1137203.

Why again, are we leaving a transition of 0s with no easing property? Seems like we could just remove it (ie. this part `background-color 0s ease, visibility 0s ease`) and achieve the same effect?